index.html

```diff
@@ -10,16 +10,41 @@
 </head>
 
 <body>
-  <h1>Awesome Books</h1>
-  <section>
-    <ul>
+    <section class = "nav-menu">
+        <nav class="navbar">
+         <h1 class="title">Awesome Books</h1>
+         <ul class="nav-list">
+             <li id ="list" class="list"> List</li>
+              <li id= "new" class="new"> Add-New</li>
+               <li id= "contact" class="contact"> Contact</li>
+         </ul>  
+        </nav>
+    </section>
+  <section id= "book-list" class ="book-list">
+      <h2 class="all-books">All awesome books</h2>
+    <ul id="books">
     </ul>
   </section>
   <section id="userInput">
+    <div class="input">
+      <h2 class= "addtitle">Add a new book</h2>
     <input id="title" type="text" placeholder="Title"  required ><br><br>
     <input id="author" type="text" placeholder="Author" required ><br><br>
     <button id="addBtn">Add</button>
+    </div>
   </section>
+  <section id="contact-us" class="contact-us">
+      <h3 class="info">Contact information</h3>
+      <p class= "para"> Do you have any questions or you just want to say "Hello"? <br> You can reach out to us!</p>
+<ul class="conta-info">
+    <li>Our e-mail: mail@gmail.com</li>
+    <li>Our phone number: 0043586534422</li>
+    <li>Our address: streetName 22,84503 City, Country</li>
+</ul>
+  </section>
+  <footer>
+      <p>Copyright  ....</p>
```

Comment:
- [ ] [OPTIONAL] Kindly let's ensure that the footer is well styled by giving it the necessary spaces as it is in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements).
![Screenshot from 2022-05-27 21-22-08](https://user-images.githubusercontent.com/58692590/170786227-fab1b5ab-cd36-4203-8e2d-5bcb6d25579d.png)